Expose `activate-pane-direction` cli command

abusch opened the pull request:
Add a new subcommand for `wezterm cli` called `activate-pane-direction`. It allows switching the active pane in the current tab in the given direction.

closes #2525

wez reviewed (commented):
Thanks for this!

Please also increment the `CODEC_VERSION` number in `codec/src/lib.rs`, and note that I just bumped it last night so you'll want to bump it up one more from what is currently in `main`!

The implementation looks reasonable, the main comment is around parsing the direction in the CLI; suggestion inline!

abusch commented on wezterm/src/main.rs:
```diff
@@ -367,6 +367,25 @@ Outputs the pane-id for the newly created pane on success"
+
+        /// Activate the pane below the current pane
+        #[clap(long, conflicts_with_all=&["left", "right", "up"])]
+        down: bool,
```
Done. Had to make it a method of `PaneDirection` directly as it needs private APIs (maybe this was implied...).

abusch commented:
Alright, I think I've addressed the feedback. I've also ran `cargo fmt` and merged `main` in to resolve the conflict. Let me know if there is anything else!
